src/Contexts/UserContext.js

```diff
@@ -0,0 +1,34 @@
+import React, { useState, createContext, useEffect } from "react";
+import { getUser } from "../apiCall";
+
+export const UserContext = createContext()
+
+const UserProvider = (props) => {
+
+    const [user, setUser] = useState({})
+
+    const [journal, setJournal] = useState([]);
+
+    const [mood, setMood] = useState([])
+
+    const values = { user, setUser, journal, setJournal, mood, setMood}
+
+    useEffect(() => {
+        getUser()
+            .then(data => setData(data))
+    }, [])
+
+    const setData = (data) => {
```

Comment:
Nice work around

Reply:
Is this logic with the useEffect used to set the initial data the API if it exists?